util/logtools/encounter_tools.ts

```diff
@@ -187,7 +190,6 @@ export class EncounterFinder {
     const netSeal = this.regex.netSeal.exec(line)?.groups;
     if (netSeal) {
       this.onNetSeal(line, netSeal.param1, netSeal);
-      this.storeStartLine(line, store);
```

Comment:
Given the changes you made to `onStartFight()`, I think this makes sense, but I'm having a not-great brain day. What's the reasoning behind leaving `storeStartLine()` in place for game log seals?

Reply:
> Given the changes you made to `onStartFight()`, I think this makes sense, but I'm having a not-great brain day. What's the reasoning behind leaving `storeStartLine()` in place for game log seals?

Yeah, so my rationale there was that we should never invoke `storeStartLine()` after `onStartFight()`, since it will simply return.  And `onNetSeal()` invokes `onStartFight()`.

I kept `storeStartLine()` because the game log seals invoke `onSeal()`, which doesn't invoke `onStartFight()`.  I couldn't figure out why that was, but I assumed there was some reason, and I didn't want to mess with existing behavior (e.g., in which `storeStartLine()` actually might do something) without fully understanding.  

Definitely open to alternatives here if you have more historical context!

Reply:
Oh, right, I can't remember the specifics behind that decision, but looking at it more, it's likely to avoid having  `onSeal()` clobber an existing `sealId` if a net seal is present. Realistically it will probably never matter for an end-user, since the timeline will inevitably be able to start, but I'll maybe come back to this later and see whether we can figure out a better way to guarantee recording seal names.